e2e_test/streaming/tpch/views/q9.slt.part

```diff
@@ -12,8 +13,8 @@ from
       l_extendedprice * (1 - l_discount) - ps_supplycost * l_quantity as amount
     from
       part,
-      supplier,
       lineitem,
+      supplier,
```

Comment:
Note that we tweak the order to pass the test. Otherwise the test will lead to CROSS Join, which is unimplemented. Track issue #1866.